src/Plug.vala

```diff
@@ -34,7 +34,7 @@ public class GalaPlug : Switchboard.Plug {
 
         // DEPRECATED
         settings.set ("desktop/wallpaper", "wallpaper");
-        
+
         Object (category: Category.PERSONAL,
```

Comment:
I guess you need to move this GObject part to the `public GalaPlug ()` constructor